samplescalaproject/src/test/scala/example/FunctionsSpec.scala

```diff
@@ -25,4 +26,16 @@ class FunctionsSpecs extends WordSpec {
         val multiplesOf = Functions.intMultiples(2)
         assert(multiplesOf == List(2,4,6,8,10,12,14,16,18))
     }
+
+       "The object Person should take variable parameters of type String and Int and yield result values in string" in {
+        val person = new Person (name = "Senior", age = 19, nationality = "Nigeria")
+ 
+        println(Person(person))
```

Comment:
but the assertion was not done . eg are you testing that it should return value of age as 19 or the nationality as "Nigeria" 

class PersonSpec extends AnyFlatSpec with Matchers {
  val (name, age, nationality) = ("Michael", 12, "Nigerian")
  val expectedString = s"Hi. I am [$name] from [$nationality], and I am [$age] years old!"

  "The Person class" should "create a person" in {
    assert(new Person(name, age, nationality).toString == expectedString)
  }

  it should "create a person without the 'new' keyword" in {
    assert(Person(name, age, nationality).toString == expectedString)
  }

  it should "not allow negative age input" in {
    an [AssertionError] should be thrownBy {
      Person(name, -1, nationality) // an alien :(
    }
  }

Reply:
Thanks Ayo for the corrections. I will update accordingly.